storage/object/reference-content/cname-record.mdx

```diff
@@ -0,0 +1,84 @@
+
+    TODO SCREENSHOT
+
+## How to create a CNAME record
+
+Log into your domain provider, and locate the DNS settings for your domain. Create a new CNAME record pointing your subdomain to the Edge Services endpoint for your bucket. This endpoint will be displayed in the Scaleway console.
+
+The interface used by different domain providers varies, but creating your CNAME record may look like one of the following examples:
+
+| Record                                     | Destination                              |
+|--------------------------------------------|------------------------------------------|
+| `my-chosen-subdomain.beautiful-domain.com` | `bucket-name.edge.scw.cloud`             |
+
+| Subdomain                                  | Target host                                    |
+|--------------------------------------------|------------------------------------------------|
+| `my-chosen-subdomain`                      | `bucket-name.edge.scw.cloud`                   |
+
+| Host record                                | Points to                                      |
+|--------------------------------------------|------------------------------------------------|
+| `my-chosen-subdomain`                      | `bucket-name.edge.scw.cloud`                   |
+
+| `my-chosen-subdomain`                      | Record Type     | Value                                          |
+|--------------------------------------------|-----------------|------------------------------------------------|
+| `@`                                        | `CNAME`         | is an alias of `bucket-name.edge.scw.cloud`    |
+
+
+You may also see a `TTL`field, which stands for **T**ime **T**o **L**ive.  This tells the DNS resolver how long it can cache this record, before it must re-check the original source in case something has changed. TTL is measured in seconds, and the default value is usually 12 hours (43200 seconds) or 24 hours (86400 seconds). 
+
+ ## Troubleshooting DNS errors
+
+When setting up your customized subdomain with Edge Services, you have the option to carry out a verification check on the CNAME record. Edge Services will query the subdomain and check that it resolves correctly to the Edge Services endpoint. If there is a problem, you will see an error message. 
+
+    TODO SCREENSHOT
+
+An error message may also display at a later point from your Edge Services dashboard if a problem is detected at any point with your CNAME record or subdomain.
+
+See the table below for help with troubleshooting these errors:
+
+| Error message                             | Solution                                                            |
+|-------------------------------------------|---------------------------------------------------------------------|
+| No CNAME record found                     | Make sure you have created a valid CNAME record where your subdomain points to the Edge Services endpoint.  |
```

Comment:
@RoRoJ, I believe the issue here is that when we try to resolve the domain name we either realize that there is no record (need to double check with Valentin as no record may be your third case), or the record we found is not a CNAME record (like we get an IP address directly because it's an A or an AAAA record).